### DISCUSSION TITLE
How to update Coqui-TTS to the latest version?

### DISCUSSION THREAD

**Discussion by arif334** [coqui-ai/TTS#935] [General Q&A] [answered]
I installed Coqui-TTS a few months ago. It was probably of version v0.1.x or similar. Since then the repository has been updated several times.

Now I want to update my local copy to the latest version of Coqui-TTS. What would be the proper way to do this, except removing and cloning the repo again? I don't want to keep the local changes.

**Comment by thorstenMueller** [comment]
Have you tried `git pull` in the cloned base directory?
Or do you mean updating the pip package? Then `pip install tts --upgrade` should do the job.

**Reply by arif334** [reply]
> Have you tried `git pull` in the cloned base directory? Or do you mean updating the pip package? Then `pip install tts --upgrade` should do the job.

`git pull` says `Already up to date`. But I'm certain that it is not, since most of the recent recipes are not present there. By the way, is there any way to find out in which version I am in?

**Comment by erogol** [comment, marked as answer]
`pip install -U tts` if you installed by `pip`

else

`git pull origin master ` is the latest released version
`git pull origin dev ` is the latest development version
